src/DSharp.Script/src/System/Collections/Interfaces.js

```diff
@@ -0,0 +1,16 @@
+// Contracts
+
+function ICollection() { }
+function ICollection_$1() { }
+function IReadOnlyCollection_$1() { }
+
+function Dictionary_$2() { }
+function IDictionary() { }
+function IDictionary_$2() { }
+function IReadOnlyDictionary_$2() { }
+
+function List_$1() { }
```

Comment:
Move to a new script file called List.js, under System/Generic/Collections folder